deps/rabbit/src/rabbit_amqqueue.erl

```diff
@@ -339,7 +339,9 @@ is_server_named_allowed(Args) ->
             [amqqueue:amqqueue()].
 
 lookup(Name) when is_record(Name, resource) ->
-    rabbit_db_queue:get(Name).
+    rabbit_db_queue:get(Name);
+lookup(Names) when is_list(Names) ->
+    lists:map(fun rabbit_db_queue:get/1, Names).
```

Comment:
For better or worse, Team RabbitMQ has historically used list comprehensions over `lists:map/2` for basic map operations

Reply:
:slightly_smiling_face: So be it, @michaelklishin.